### DISCUSSION TITLE
Is there any free machine translation option better than Google Translate V1?

### DISCUSSION THREAD

**Discussion by angelfire76** [SubtitleEdit/subtitleedit#8131] [General]
The current options that require an API require a paid account as far as I can see. Is there a free model that does a better job than GT V1?

**Comment by niksedk** [comment]
You can use Google Translate V1 with a free quota each day.
Another option is DeepL free API (requires enabling of billing). Is probably a tiny bit better than Google Translate.

Also, more and more "Chat GPT" like models are coming all the time. Many you can run locally via "LM Studio" or Ollama, which will be supported in next SE version. These models are slow (unless you have a nice gfx card), and the quality is probably yet worse then Google Translate.

Libre translate is also okay, but still not as good as GT.

**Reply by ghost** [reply]
There is [DeepLx ](https://github.com/OwO-Network/DeepLX)that you can use DeepL free api without having account.

**Reply by smalhotr85** [reply]
how much Google Translate V1 with a free quota each day?

**Comment by gericho** [comment]
I got excellent and relatively fast translations with nllb-serve and [facebook/nllb-200-distilled-1.3B,](https://huggingface.co/facebook/nllb-200-distilled-1.3B) there is also a [3.3B](https://huggingface.co/facebook/nllb-200-3.3B) version, LLM like Hermes 2 Pro and openhermes, are more colloquial language aware than NLLB, but worse in grammar. From Spanish to Sinhala, the readers refer the translation is almost perfect, way more grammar-corrected compared to Google V1! I'm not sure it's an exception with this language combo, but I advice anyone with the necessary hardware requirements to try it and give a feedback. Thank you!

EDIT: Actually, after some more testing, Google V1 is still the king, NLLB come second. LLMs I tried are still giving garbage output with these language pairs, BUT still very good for Latin-only base languages

**Reply by dgoryeo** [reply]
@gericho, thanks for the pointer. Would you have a sample code on or a notebook?

**Comment by gericho** [comment]
I checked today with mother-tongue people, they say 3.3B is slightly better than 1.3B. Be aware 3.3B would eventually run only on the CPU so painfully slow, whereas the 1.3B with 8GB VRAM runs way faster, not on par with an online service like Google V1 but you can get 2h translation in 5 minutes or so. The Google V1 btw is still superior to NLLB but only on some specific parts of the text. Keep in mind I can talk only for the language pair just mentioned above, so your mileage may vary. 
About the guide, I just followed the Subtitle Edit FAQ in the NLLB section [here](https://www.nikse.dk/subtitleedit/help#translation)

Just a side note, I think there will be more chances in using an LLM Studio approach and a properly trained model for a specific language pair from Huggingface for example, unfortunately, fine-tuned models are only a few.

**Comment by gericho** [comment]
I'd like to bring your attention to a very well-made and up-to-date model comparison by [Wolfram Ravenwolf](https://www.reddit.com/user/WolframRavenwolf/submitted/)

I am testing the top-rated models and see how they perform.

**Comment by gilbrotheraway** [comment]
for brazilian portuguese the only viable option is ollama, google api's i'm not sure how to setup but if it's anything like translate.google.com it's worthless

**Reply by AuroraMartell** [reply]
> for brazilian portuguese the only viable option is ollama, google api's i'm not sure how to setup but if it's anything like translate.google.com it's worthless

I use ChatGPT or Gemini for translation.

**Comment by Safiralilas** [comment]
I haven't found a better translator than Google yet, V1 is on SE and V2 is acquiring the API key, but......copying and coloring gives you the chance to use V2 for free, SE does this for you, copy and paste on the website, copy and paste, copy and copy and paste and THAT'S IT, the best translation, if you take care when putting together the subtitles, apply good spelling and grammar, there's no mistake, it's a success.

**Comment by FrankenHu** [comment]
End of 2025. Any new recommendations?

**Reply by drHyperion451** [reply]
Tried ollama with the llama 3.2 and give me worse results than Google Translate v1. I will suggest using this and fixing some translating mistakes on the fly.

**Reply by hundredfire** [reply]
I've had great results using the "auto-translate via copy-paste" and gemini pro. It even works with SSA tags. It's quite tedious of a process though. I've seen some scripts that were used to automate the process for .srt files. It might be trivial to adapt it to .ssa files but I haven't tried it

**Comment by xindex2** [comment]
Hey, I know exactly what you mean, finding a free translation API that’s better than Google Translate V1 is tough. 
Most hosted APIs are paid, and the open-source/free models often lag in quality or require self-hosting and GPU resources.

If you want a reliable, developer-friendly option that’s cheaper than the big providers, I built **[https://translate-api.com/](https://translate-api.com)**. 
It uses modern neural translation models, supports 100+ languages, and works via a simple REST API. 

It’s not fully free, but the entry cost is very low, and it delivers quality comparable to Google Translate without needing heavy infrastructure.

For experiments or small projects, it’s a good middle ground between free but limited models and expensive commercial APIs.

**Reply by HaraldBluetooth63** [reply]
For translations between European languages, I haven't yet found a better translation api, than DeepL. It gives fluent translations with correct word order and really good for nuances in dialogue and subtitle tone. I've a free account with a 500,000 characters limit per month.
